app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Far Mar | Alice &amp; Ashley</title>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= csrf_meta_tags %>
+</head>
+<body>
+
+<header>
+  <nav>
+    <%= render "layouts/#{@nav}_nav" %>
```

Comment:
This is a bit of a smell to me. I get that you did it because you don't want to put your header and nav tags inside your view for each page, but this line seems pretty fragile and easy to break by another developer not as familiar with your code. Typically I see navs that have different links in them depending on the current user's permissions, but otherwise the nav doesn't change radically depending on the view. It might read better if you had a single partial for your nav and pass in a variable that would indicate whether you should hide or show different links.